src/Model/Changelog/ChangelogItem.php

```diff
@@ -70,17 +70,17 @@ public function setChangelogLibrary($changelogLibrary)
     protected static $types = array(
         'Security' => array(
             // E.g. "[ss-2015-016]: Security fix"
-            '/^(\[SS-2(\d){3}-(\d){3}\])\s?:?/i'
+            '/^(\[SS-2(\d){3}-(\d){3}\])\s?:?\s+/'
```

Comment:
All these regex expressions could be simplified by removing the `\s?:?\s+` and just replacing with `\b` (word boundary)

Reply:
Implemented in each rule other than this security one where word boundary matching didn't work (probably because of the brackets)